frontend/src/Store/Actions/movieTitlesActions.js

```diff
@@ -0,0 +1,74 @@
+import { batchActions } from 'redux-batched-actions';
+import createAjaxRequest from 'Utilities/createAjaxRequest';
+import { createThunk, handleThunks } from 'Store/thunks';
+import createHandleActions from './Creators/createHandleActions';
+import { set, update } from './baseActions';
+
+//
+// Variables
+
+export const section = 'movieTitles';
+
+//
+// State
+
+export const defaultState = {
+  isFetching: false,
+  isPopulated: false,
+  error: null,
+  items: []
+};
+
+//
+// Actions Types
+
+export const FETCH_MOVIE_TITLES = 'movieTitles/fetchMovieTitles';
```

Comment:
Doesn't look like we use this anywhere, Assume this is for future use if we decide to breakup titles from movies?

Reply:
Yes, this was for future use with /alttitle endpoint.